llvm/lib/DebugInfo/DWARF/DWARFUnit.cpp

```diff
@@ -1051,6 +1051,17 @@ DWARFUnit::getLastChildEntry(const DWARFDebugInfoEntry *Die) const {
   return nullptr;
 }
 
+const DWARFAbbreviationDeclaration *DWARFUnit::tryExtractCUAbbrevFast() const {
+  Expected<const DWARFAbbreviationDeclaration *> AbbrevOrError =
+      Abbrev->tryExtractCUAbbrevFast(getAbbreviationsOffset());
+  if (!AbbrevOrError) {
+    // FIXME: We should propagate this error upwards.
```

Comment:
Why not have this return an `Expected` as well? You added this function so you get to decide how its used, no?